app/server/api/suggestion/[name].post.ts

```diff
@@ -0,0 +1,26 @@
+import type { CompatibilityEvent } from 'h3'
+import { defineEventHandler } from 'h3'
+import { parseParamsAs, z } from '@sidestream-tech/nuxt-sidebase-parse'
+import { format } from 'date-fns'
+import { Suggestion } from '~/server/database/entities/Suggestion'
+
+const paramsSchema = z.object({
+  name: z.string().min(1),
+})
+
+export default defineEventHandler(async (event: CompatibilityEvent) => {
+  // const SUGGESTIONS_FOLDER = isProduction ? './suggestions' : '.'
+  // const filePath = `${SUGGESTIONS_FOLDER}/suggestions.log`
+  // fs.appendFileSync(filePath, `${now}; ${params.name}\n`)
+
+  const params = await parseParamsAs(event, paramsSchema)
+  const now = format(new Date(), 'yyyy-MM-dd HH:mm:ss')
+
+  const payload = { name: params.name, date: now }
+
+  console.log(params.name)
```

Comment:
and I think this can either go OR should be extended to be more informative, e.g., with a time?